fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogBottomSheet.kt

```diff
@@ -262,43 +311,120 @@ private fun TranslationsDialogInfoMessage(
     }
 }
 
+@Suppress("MagicNumber")
 @Composable
 private fun TranslationsDropdown(
     header: String,
     modifier: Modifier = Modifier,
+    isInLandscapeMode: Boolean = false,
+    translateLanguages: List<Language>,
```

Comment:
Let's move the required parameters before the optional parameters. This would follow the API guidelines described in https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-component-api-guidelines.md#parameters-order.